instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/sqs.py

```diff
@@ -24,3 +29,16 @@ def extract_attributes(self, attributes: _AttributeMapT):
         if queue_url:
             # TODO: update when semantic conventions exist
             attributes["aws.queue_url"] = queue_url
+
+    def on_success(self, span: Span, result: _BotoResultT):
+        operation = self._call_context.operation
+        if operation in SUPPORTED_OPERATIONS:
+            span.set_attribute(SpanAttributes.MESSAGING_SYSTEM, "aws.sqs")
```

Comment:
I _think_ there should be a constant for this value too in `SpanAttributes`

Reply:
Nope, there isn't. Only for dynamo and lambdas.
I can add there, but it will take ages to be approved